src/component/common/ConstraintAccordion/ConstraintAccordionView/ConstraintAccordionViewHeader/ConstraintAccordionViewHeader.tsx

```diff
@@ -46,46 +80,49 @@ export const ConstraintAccordionViewHeader = ({
             onDelete();
         });
 
+    constraint.contextName = 'verylonglonglonglongcontextname';
```

Comment:
This was for testing I presume? Currently is crashing the app on vercel because it's trying to assign to a read only variable when adding a constraint.

Reply:
You're right of course, forgot about this.